src/main.c

```diff
@@ -2624,33 +2629,61 @@ void render(void){
     glMatrixMode(GL_MODELVIEW);
     glLoadIdentity();
 
+    glPushMatrix();
+    glLoadIdentity();
+    glColor3f(1.0f,1.0f,1.0f);
+    glRasterPos2f(0.46, 0.90);
+    glutBitmapString(GLUT_BITMAP_TIMES_ROMAN_24, "CRAFT");
+    glPopMatrix();
+
     glPushMatrix();
     glTranslatef(0.0, 0.1, 0.0);
-    display();
+    renderButtonBackground();
+    glPopMatrix();
+
+    glPushMatrix();
+    glLoadIdentity();
+    glColor3f(1.0f,1.0f,1.0f);
+    glRasterPos2f(0.45, 0.76);
+    glutBitmapString(GLUT_BITMAP_TIMES_ROMAN_24, "Play Game");
```

Comment:
This explains why the menu pops up at the start of main.  Sorry for missing that earlier!